example/secure_client.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+
```

Comment:
Personally I find that adding a one-line comment to repeat the purpose of the file
could help a bit; in this particular case something like

    # the client side that runs over wss / TLS